### PR TITLE
Display the tool's stderr/stdout on build failure

### PR DESCRIPTION
**Pull request by Kernald**

### CODE REVIEW

**Comment by Kernald** [comment]
@solsjo I think this was a regression in #84 - mind taking a look?

File: tool_wrapper.py [hunk 1]
```diff
@@ -89,12 +89,17 @@ def main():
...
+    result = ""
+    try:
+        result = subprocess.check_output(
+            args=cmd_args,
+            env=env,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True,
+        )
+    except subprocess.CalledProcessError as exc:
+        print("Failed to run tool: ", exc.returncode, exc.output)
```
Comment (solsjo) [inline tool_wrapper.py]: LGTM

But maybe for consistency.

Would it make sense to add the print out to the sys exit below?
With add an additional condition for success of the above call to the if-statement.

Or do we expect there to exist failures that we want to print but still generates the output?

**Comment by solsjo** [comment]
Ack! Looks great. I don't know why I hid that.

Had a single review comment.

**Comment by solsjo** [comment]
@Kernald Fixing the review comment is not mandatory though, feel free to bring it in if you disagree.

File: tool_wrapper.py (hunk 1 above)
Comment (Kernald) [reply tool_wrapper.py]: Good point! Done. I'm also checking the return code, so even if somehow an output is produced but a return code != 0 is returned, this should fail in the same way.

**Pull request merged**